gui/src/components/s3/bucket-creation.vue

```diff
@@ -488,6 +488,7 @@ export default class CortxBucketCreation extends Vue {
         apiRegister.bucket_policy + "/" + bucketname
       );
       this.policyJSON = JSON.stringify(res.data, null, 4);
+      this.noBucketPolicy = false;
```

Comment:
what will happen if policy was not set previously. I think still it will enable Delete policy button which is wrong

Reply:
![image](https://user-images.githubusercontent.com/66409360/113868057-e4a3e180-97cc-11eb-9a38-34b6a915df17.png)
if bucket policy is not set previously delete button will disable.